jenkins/shared-library/vars/githubHelper.groovy

```diff
@@ -29,12 +25,8 @@ String executeGithubApiRequest(String command) {
 	String results = ''
 	helper.withTempDir {
 		String file = "./${System.currentTimeMillis()}"
-		final String response = runScript("${command} -s -w '%{http_code}\\n' -o ${file}", true).trim()
+		runScript("${command} -s -o ${file}")
```

Comment:
there is exception on failure now?

Reply:
There should  be an exit code but still need to verify